pontoon/base/templatetags/helpers.py

```diff
@@ -1,8 +1,9 @@
 import cgi
 import datetime
 import json
-import urllib
-import urlparse
+
+from six import text_type
+from six.moves.urllib import parse as urlparse
```

Comment:
Six's documentation shows that all methods that were in python 2 `urlparse` are supported in `six.moves.urllib.parse` and also there is no such library as urlparse in python 3 (became `urllib.parse`). I think it will be fine to use urlparse name.